.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@0.1.6
+  plugin-ci: mattermost/plugin-ci@0.1.12
```

Comment:
Why not use `@volatile`?

Reply:
I noticed most (all?) other plugins have `volatile`. Do you manually set this value on the other plugins? If so, how do you make sure this is done?

Reply:
@hanzei I want to make sure I understand the process of updating the orb version in the individual plugins. Is this something that's done informally at some point after plugin in-take/creation?

Reply:
Exactly. See https://github.com/mattermost/mattermost-plugin-splunk/commit/bf52330652f460751371db36c67eeb7cca4bbd70 for an example. Without these changes the release pipeline would not even work. So that makes sure they get done